pages/signup.html

```diff
@@ -0,0 +1,65 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="/css/reset.css" />
+    <link rel="stylesheet" href="/css/style.css" />
+    <link rel="stylesheet" href="/css/signin.css" />
+    <title>판다마켓</title>
+  </head>
+  <body>
+    <main class="wrap">
+      <a href="/" class="logo">
+        <img src="../images/logo.svg" alt="판다마켓" />판다마켓
+      </a>
+      <form>
+        <label for="email">이메일</label>
+        <input
+          type="email"
+          name="email"
+          id="email"
+          placeholder="이메일을 입력해 주세요"
+        />
+        <label for="nickname">닉네임</label>
+        <input
+          type="text"
+          name="nickname"
+          id="nickname"
+          placeholder="닉네임을 입력해 주세요"
+        />
+        <label for="pwd">비밀번호</label>
+        <input
+          type="password"
+          name="pwd"
+          id="pwd"
+          placeholder="비밀번호를 입력해 주세요"
+        />
+        <div class="ico_hide">
+          <img src="../images/ico_hide_pwd.svg" alt="" />
+        </div>
+        <label for="re_pwd">비밀번호 확인</label>
+        <input
+          type="password"
+          name="re_pwd"
+          id="re_pwd"
+          placeholder="비밀번호를 다시 한 번 입력해 주세요"
+        />
+        <div class="ico_hide">
+          <img src="../images/ico_hide_pwd.svg" alt="" />
+        </div>
+        <button type="submit">회원가입</button>
+      </form>
+      <div class="quick_signin">
+        <span>간편 로그인하기</span>
+        <a href="https://www.google.com/" class="google" target="_blank"></a>
+        <a
+          href="https://www.kakaocorp.com/page/"
+          class="kakao"
+          target="_blanks"
+        ></a>
```

Comment:
self-closing tag 사용하셔도 좋을 것 같아요! 

```suggestion
        <a href="https://www.google.com/" class="google" target="_blank" />
        <a
          href="https://www.kakaocorp.com/page/"
          class="kakao"
          target="_blanks"
        />
```